firebase-firestore/src/main/java/com/google/firebase/firestore/DocumentId.kt

```diff
@@ -37,11 +38,6 @@ import kotlinx.serialization.SerialInfo
  */
 @SerialInfo
 @Retention(RetentionPolicy.RUNTIME)
```

Comment:
please remove retention, it defaults to runtime in kotlin, if you prefer to keep it, please switch it to kotlin.Retention